cypress/integration/chi-custom-elements/button.spec.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+
+describe('Buttons', () => {
+
+  it('Base button should be included inside the custom element', () => {
+    cy.visit('tests/custom-elements/buttons.html');
+
+    cy.get('[data-cy="button"]')
+      .wait(5000)
+      .first()
+      .should('have.class', 'hydrated')
+      .children()
+      .first()
+      .should('match', 'button.a-btn');
+
+  });
+
+  it('Colored buttons should have appropriate class', () => {
+    cy.get('[data-cy="button"]')
+      .eq(1)
```

Comment:
is it right?

Reply:
Yeah instead of getting first element from the list I'm picking the second one.